docs/reference/continuous-export.md

```diff
@@ -0,0 +1,169 @@
+object store using its native paths and access methods.  This allows clients and tools
+read-only access to repository objects without any lakeFS-specific configuration.
+Initially, only AWS S3 is supported.
+
+For instance, the contents `lakefs://example@master` might be stored on
+`s3://company-bucket/example/latest`.  Clients entirely unaware of lakeFS could use that
+base URL to access latest files on `master`.  Clients aware of lakeFS can continue to use
+the lakeFS S3 endpoint to access repository files on `s3://example/master`, as well as
+other versions and uncommitted versions.
+
+A one-time export copies all objects on a branch to a location on S3, once.  A continuous
+export tracks the latest state of a branch on a location in S3: it updates that location
+in S3 after each change in lakeFS (either a commit or a merge)
+
+## Configuration and management
+
+Configuration and management are performed via a CLI that calls a simple REST API on the
+lakeFS server.  Either can be used.  You may configure continuous export on any number of your
+branches.
+
+### CLI
+
+The command `lakectl export` is the entrypoint to all export commands.
+
+#### `lakectl export get`
+
+Read the current export configuration for a branch using
+```shell
+lakectl export get lakefs://REPO@BRANCH
+```
+
+For instance,  `./lakectl export get lakefs://foo@master`.
+
+#### `lakectl export set`
+
+Set an export configuration for a branch by using
+```shell
+lakectl export set lakefs://REPO@BRANCH FLAGS...
+```
+
+Use these flags:
+* `--continuous`: enables _continuous export_, after every commit or merge to the branch.
+  Must be specified, so set `--continuous=false` to immediately start one single export.
+* `--path`: specifies the destination path to export to.
+* `--status-path`: specifies the export status path, under which a file is written giving the
+  status after each export.
+* `--prefix-regex`: identifies prefixes (directories) in which to place "export done" objects
+  (useful for triggering workflows).  May be repeated to specify multiple regular expressions.
+  
+#### `lakectl export run`
+
+Start exporting the requested branch now by using
+```shell
+lakectl export run lakefs://REPO@BRANCH
+```
+
+#### `lakectl export repair`
+
+If an export fails, for instance if the daemon has no permissions to write to the export
+directory, lakeFS has no way of determining how best to proceed or when to try to proceed.
+After repairing the issue, e.g. by granting permissions, restart by using
+```shell
+lakectl export repair lakefs://REPO@BRANCH
+```
+
+### API
+
+#### Configuration
+
+Configure and query continuous export configuration of a branch with a `PUT` or a `GET` to
+`/repositories/{repository}/branches/{branch}/continuous-export`.  Pass these JSON
+parameters in the body:
+```json
+{
+  "exportPath": "s3://company-bucket/path/to/export", // export objects to this path (required)
```

Comment:
AFAIK Go's JSON decoder will fail if passed a string containing comments. Many people will copy this verbatim and edit the parts they care about, which I believe will lead to an error...

Reply:
```json
{
    "status": "fixed"     ,   
  "message": "I <3 JSON."
}
```